python/rmm/docs/conf.py

```diff
@@ -197,7 +197,10 @@
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3", None),
     "numba": ("https://numba.readthedocs.io/en/stable", None),
-    "cuda-python": ("https://nvidia.github.io/cuda-python/", None),
+    "cuda-python": (
+        "https://nvidia.github.io/cuda-python/cuda-bindings/",
+        None,
+    ),
```

Comment:
CI is blocked in this repo, because docs builds are failing like this:

> /__w/rmm/rmm/python/rmm/docs/docstring of rmm.pylibrmm.device_buffer.DeviceBuffer.prefetch:10: WARNING: py:attr reference target not found: cuda.bindings.runtime.cudaCpuDeviceId

([build link](https://github.com/rapidsai/rmm/actions/runs/11811388149/job/32905156338#step:10:1284))

@harrism pushed a fix for that to #1724 (https://github.com/rapidsai/rmm/pull/1724/commits/5330063f6463fe05263e114a18fb5d1b9c3c4147), but I'm not sure how close that PR is to being merged.

Proposing the same fix here, in this ready-to-merge PR, to help unblock CI.